src/cli/search.rs

```diff
@@ -99,11 +100,155 @@ pub async fn execute(args: Args) -> miette::Result<()> {
     let repo_data = fetch_sparse_repodata(&channels, &platforms).await?;
 
     let p = PackageName::try_from(package_name.clone()).into_diagnostic()?;
+
```

Comment:
As the code got some more paths now it would be nice to have a comment in the code that explains the flow of the paths.